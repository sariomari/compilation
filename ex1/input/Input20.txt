6646
* 9
W*[-(3

4
//415
9287
(nil94
}nil]*[-]//;K}}245
nil}739
*+	""14
++41
)]d+nil54
"")+43
]"WkqpN"6
*]//T}8B
9475
[+*)--{""5887
/*]
}]?
*/*
//
)("XN"	])M8[nil[)*nil"MbOKzUW")
{//??u[+5
(*+[[*5
50
+8923
+nil{[//?;KTI?}{IVkJ;}!?;I]+[//
A422
3209
k
5545
8
{[(/*;]*/{/*EIY}
a/?]s];d?k??;?};*/]705
nil-{/*]}]Q}8!1g)k)3]pY]}C?h;!T61!]}(*/(/**//**/[*]})[fLO3]
72
//76

nil }{
*[/*;*///]M]R(7
-{**]{2862
]//}p]Hv![)-	79
)(645
}//.3Kp(!51
9999
//A;}vr];g
nil([*2
 -+{}[(*nil){]"HW"}""nil	2183
)nil9241
---nil7
	"Yz"0
]
"" 895
	*}9
*]*}*)}"eQCn"30
}""759
()-(gcH)""/*
3}
7
;(i](?!
s;!?Lw}!}7g?;l8l)?}6s!Z]?]1!]!!)2?m;X)}}lY}!95!])?}]?
;*//*}}?)})}*/-{9
] 9061
{292
20
*	8690
- 6
{3383

1
+[)nilnil//)
(-)[{2
/**/+89
541
//]}!}536
	"R"++44
31
	+nil""]R}29
({//nil
*]6775
1171
[{*0
21
)271
EQU2iNf}{]*nil1736
]})//4427
""]]""[h4E9CE3I9Y2[{**+"c"
(2252
7
{*	-//])!6?!
{- +
]RR))//!})96
 )9291
(*	*""{VV+)47
[529

-]nilD+ })	-*[ ([""(	9287
"")""*585
*+nilkVW35Gb11l0t//?H [*{6
({Bp3S)60
"ySIJbPPJgY"8

nil2017
nil//;;/*H;(
?)?v]Etu].{
P?]};;}(}?N?!!a(K*/{[-5
"m""C"-2853
/*;];!
*/13
]42
-{)//
"U"885
/*(8}(
}*/252
//l!}}s]X(
(5
+4
662
-417
-""}1
++62
5
8056
99
](825
k2r{/*]}}EO
*/ ](/*
2?
)*/+{-"ViO"/*0;]
}X}(
?
}}j!;(;*/	)7
//*/*e
?
)]S*/)/*
})?;u*/[){]aWAU[)}{[/*n}*///;;;3(6

{//;];k0}36egr]})[)//?]}}{""19
	]}"W"++) 	5
//7S
+57
80
-40
//!
21
14
(//6048
P}974

(]897
4731
}]j{([[nN//;;?b88
+( 3
nil}26
)])33
2215
}34
)/*(2H;}};;]!U?8(i]
!}};!};Z;};]o?!(}]!]dQkGk/(?t!

][0h*/}nil+](""{/*u*/*} //}]w}48
34
){][nilnil//?1
-+SyAj7
//?)?722
nil/**/+[{-ib -871
4
})[*-)-680
(-W247
672
*-+(1977
}u//!
){785
]+ [("BF"-/*;*/)95
[""nil""/*
;!*/-[//}
{[+54
(3
 DuzCc {nil11
4
//?!]?)?})"clnLhfdJEn"""//)nil+-/*};1G?Ib}}};)}]Bk2*/*[9
-nil+10
""//(!Ct)?"orA")10
//;?}}9
gq//354
4195
]69
{]]40
d}
]+680
76
67
+""{ ])]+*)(
2851
-*nil1412
291
"SvK"90
-3314
1
/*.*/4157
*-}Ycod*){9792
[}4
//8
)""]"KTRo"
//VU!nil+ ""(]-*"JvF"()/**/)]/**/-"Sz"40
)(nil)(5552
+]566
//t;;!
){	*/*m]T};]7*///!!!({)w
 //]((
*14
)574
nil395
)-(- }"")}
/*;;?]Z8
)G}(;!)}[2!V!
(!fW;;U4(!?}
3!z5!z?1K}*/58
-)(+4183
/*)o!]}*/+}//;?]?//;!!;;w8
nil)6442
NS}""9927
}]))1
}({}(}}-"zEttCf"/*Qk
o9?*/	77
-*-*/*?!*/*[z/*7!]]}}}]!b}!!?0)(
*//*}?O;u
?v0;)*/""66
[nil953
1
nil18
-B90vm]8454
K4
400
[nil1776
1234
288
{}	//!?)()-{+L4o4-*//n]L2]]p6
{/**/{/*
2?5
}X{;22*/(980
]/*j}?*/(/*])W
!*/{
6412
//""-//;}( 326
"")]}(+{//K*{nile1*}}nil2
//(
[1278
/**/{32
 {)(]])]---237
/*;q!x
;fg!/P!*/(62
]//]E0)[(
/**/ (+ Zkj5s73
-*8

[()41
"byXOxJRvARMGbyY"x-
28
""//}?4
//
]0
+/*/V!*/73
 *70
{"HKhuI"362
123
/*
?
!m]!]*/[+//]t?;T;];H!?;c;}u})7;}?(+/*(g2;
??*/(