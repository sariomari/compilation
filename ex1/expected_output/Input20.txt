INT(6646)[1,1]
TIMES[2,1]
INT(9)[2,3]
ID(W)[3,1]
TIMES[3,2]
LBRACK[3,3]
MINUS[3,4]
LPAREN[3,5]
INT(3)[3,6]
INT(4)[5,1]
INT(15)[7,1]
INT(9287)[8,1]
LPAREN[9,1]
ID(nil94)[9,2]
RBRACE[10,1]
NIL[10,2]
RBRACK[10,5]
TIMES[10,6]
LBRACK[10,7]
MINUS[10,8]
RBRACK[10,9]
ID(K)[11,1]
RBRACE[11,2]
RBRACE[11,3]
INT(245)[11,4]
NIL[12,1]
RBRACE[12,4]
INT(739)[12,5]
TIMES[13,1]
PLUS[13,2]
STRING("")[13,4]
INT(14)[14,1]
PLUS[15,1]
PLUS[15,2]
INT(41)[15,3]
RPAREN[16,1]
RBRACK[16,2]
ID(d)[16,3]
PLUS[16,4]
ID(nil54)[16,5]
STRING("")[17,1]
RPAREN[17,3]
PLUS[17,4]
INT(43)[17,5]
RBRACK[18,1]
STRING("WkqpN")[18,2]
INT(6)[18,9]
TIMES[19,1]
RBRACK[19,2]
INT(9475)[20,1]
LBRACK[21,1]
PLUS[21,2]
TIMES[21,3]
RPAREN[21,4]
MINUS[21,5]
MINUS[21,6]
LBRACE[21,7]
STRING("")[21,8]
INT(5887)[21,10]
TIMES[25,3]
RPAREN[27,1]
LPAREN[27,2]
STRING("XN")[27,3]
RBRACK[27,8]
RPAREN[27,9]
ID(M8)[27,10]
LBRACK[27,12]
NIL[27,13]
LBRACK[27,16]
RPAREN[27,17]
TIMES[27,18]
NIL[27,19]
STRING("MbOKzUW")[27,22]
RPAREN[27,31]
LBRACE[28,1]
LBRACK[29,1]
PLUS[29,2]
INT(5)[29,3]
LPAREN[30,1]
TIMES[30,2]
PLUS[30,3]
LBRACK[30,4]
LBRACK[30,5]
TIMES[30,6]
INT(5)[30,7]
INT(50)[31,1]
PLUS[32,1]
INT(8923)[32,2]
PLUS[33,1]
NIL[33,2]
LBRACE[33,5]
LBRACK[33,6]
PLUS[34,1]
LBRACK[34,2]
ID(A422)[35,1]
INT(3209)[36,1]
ID(k)[37,1]
INT(5545)[38,1]
INT(8)[39,1]
LBRACE[40,1]
LBRACK[40,2]
LPAREN[40,3]
LBRACE[40,10]
RBRACK[45,1]
INT(705)[45,2]
NIL[47,1]
MINUS[47,4]
LBRACE[47,5]
LPAREN[50,9]
LBRACK[50,18]
TIMES[50,19]
RBRACK[51,1]
RBRACE[51,2]
RPAREN[51,3]
LBRACK[51,4]
ID(fLO3)[51,5]
RBRACK[51,9]
INT(72)[52,1]
INT(76)[55,1]
NIL[57,1]
RBRACE[57,5]
LBRACE[57,6]
TIMES[58,1]
LBRACK[58,2]
MINUS[59,1]
LBRACE[59,2]
TIMES[59,3]
TIMES[59,4]
RBRACK[59,5]
LBRACE[59,6]
INT(2862)[59,7]
RBRACK[60,1]
LBRACK[61,1]
RPAREN[61,2]
MINUS[61,3]
INT(79)[61,5]
RPAREN[62,1]
LPAREN[62,2]
INT(645)[62,3]
RBRACE[63,1]
INT(51)[65,1]
INT(9999)[66,1]
NIL[68,1]
LPAREN[68,4]
LBRACK[69,1]
TIMES[69,2]
INT(2)[69,3]
MINUS[70,2]
PLUS[70,3]
LBRACE[70,4]
RBRACE[70,5]
LBRACK[70,6]
LPAREN[70,7]
TIMES[70,8]
NIL[70,9]
RPAREN[70,12]
LBRACE[70,13]
RBRACK[70,14]
STRING("HW")[70,15]
RBRACE[70,19]
STRING("")[70,20]
NIL[70,22]
INT(2183)[70,26]
RPAREN[71,1]
ID(nil9241)[71,2]
MINUS[72,1]
MINUS[72,2]
MINUS[72,3]
ID(nil7)[72,4]
STRING("Yz")[73,2]
INT(0)[73,6]
RBRACK[74,1]
STRING("")[75,1]
INT(895)[75,4]
TIMES[76,2]
RBRACE[76,3]
INT(9)[76,4]
TIMES[77,1]
RBRACK[77,2]
TIMES[77,3]
RBRACE[77,4]
TIMES[77,5]
RPAREN[77,6]
RBRACE[77,7]
STRING("eQCn")[77,8]
INT(30)[77,14]
RBRACE[78,1]
STRING("")[78,2]
INT(759)[78,4]
LPAREN[79,1]
RPAREN[79,2]
MINUS[79,3]
LPAREN[79,4]
ID(gcH)[79,5]
RPAREN[79,8]
STRING("")[79,9]
MINUS[91,3]
LBRACE[91,4]
INT(9)[91,5]
RBRACK[92,1]
INT(9061)[92,3]
LBRACE[93,1]
INT(292)[93,2]
INT(20)[94,1]
TIMES[95,1]
INT(8690)[95,3]
MINUS[96,1]
INT(6)[96,3]
LBRACE[97,1]
INT(3383)[97,2]
INT(1)[99,1]
PLUS[100,1]
LBRACK[100,2]
RPAREN[100,3]
ID(nilnil)[100,4]
LPAREN[101,1]
MINUS[101,2]
RPAREN[101,3]
LBRACK[101,4]
LBRACE[101,5]
INT(2)[101,6]
PLUS[102,5]
INT(89)[102,6]
INT(541)[103,1]
INT(536)[105,1]
STRING("R")[106,2]
PLUS[106,5]
PLUS[106,6]
INT(44)[106,7]
INT(31)[107,1]
PLUS[108,2]
NIL[108,3]
STRING("")[108,6]
RBRACK[108,8]
ID(R)[108,9]
RBRACE[108,10]
INT(29)[108,11]
LPAREN[109,1]
LBRACE[109,2]
NIL[110,1]
TIMES[111,1]
RBRACK[111,2]
INT(6775)[111,3]
INT(1171)[112,1]
LBRACK[113,1]
LBRACE[113,2]
TIMES[113,3]
INT(0)[113,4]
INT(21)[114,1]
RPAREN[115,1]
INT(271)[115,2]
ID(EQU2iNf)[116,1]
RBRACE[116,8]
LBRACE[116,9]
RBRACK[116,10]
TIMES[116,11]
ID(nil1736)[116,12]
RBRACK[117,1]
RBRACE[117,2]
RPAREN[117,3]
INT(4427)[118,1]
STRING("")[119,1]
RBRACK[119,3]
RBRACK[119,4]
STRING("")[120,1]
LBRACK[120,3]
ID(h4E9CE3I9Y2)[120,4]
LBRACK[120,15]
LBRACE[121,1]
TIMES[121,2]
TIMES[121,3]
PLUS[121,4]
STRING("c")[121,5]
LPAREN[122,1]
INT(2252)[122,2]
INT(7)[123,1]
LBRACE[124,1]
TIMES[124,2]
MINUS[124,4]
LBRACE[125,1]
MINUS[125,2]
PLUS[125,4]
RBRACK[126,1]
ID(RR)[126,2]
RPAREN[126,4]
RPAREN[126,5]
INT(96)[127,1]
RPAREN[128,2]
INT(9291)[128,3]
LPAREN[129,1]
TIMES[129,2]
TIMES[129,4]
STRING("")[129,5]
LBRACE[129,7]
ID(VV)[129,8]
PLUS[129,10]
RPAREN[129,11]
INT(47)[129,12]
LBRACK[130,1]
INT(529)[130,2]
MINUS[132,1]
RBRACK[132,2]
ID(nilD)[132,3]
PLUS[132,7]
RBRACE[133,1]
RPAREN[133,2]
MINUS[133,4]
TIMES[133,5]
LBRACK[133,6]
LPAREN[133,8]
LBRACK[133,9]
STRING("")[133,10]
LPAREN[133,12]
INT(9287)[133,14]
STRING("")[134,1]
RPAREN[134,3]
STRING("")[134,4]
TIMES[134,6]
INT(585)[134,7]
TIMES[135,1]
PLUS[135,2]
ID(nilkVW35Gb11l0t)[135,3]
LBRACK[136,2]
TIMES[136,3]
LBRACE[136,4]
INT(6)[136,5]
LPAREN[137,1]
LBRACE[137,2]
ID(Bp3S)[137,3]
RPAREN[137,7]
INT(60)[137,8]
STRING("ySIJbPPJgY")[138,1]
INT(8)[138,13]
ID(nil2017)[140,1]
NIL[141,1]
LBRACE[145,20]
LBRACK[145,21]
MINUS[145,22]
INT(5)[145,23]
STRING("m")[146,1]
STRING("C")[146,4]
MINUS[146,7]
INT(2853)[146,8]
INT(13)[148,3]
RBRACK[149,1]
INT(42)[149,2]
MINUS[150,1]
LBRACE[150,2]
RPAREN[150,3]
STRING("U")[151,1]
INT(885)[151,4]
INT(252)[154,4]
LPAREN[156,1]
INT(5)[156,2]
PLUS[157,1]
INT(4)[157,2]
INT(662)[158,1]
MINUS[159,1]
INT(417)[159,2]
MINUS[160,1]
STRING("")[160,2]
RBRACE[160,4]
INT(1)[160,5]
PLUS[161,1]
PLUS[161,2]
INT(62)[161,3]
INT(5)[162,1]
INT(8056)[163,1]
INT(99)[164,1]
RBRACK[165,1]
LPAREN[165,2]
INT(825)[165,3]
ID(k2r)[166,1]
LBRACE[166,4]
RBRACK[168,4]
LPAREN[168,5]
PLUS[170,4]
LBRACE[170,5]
MINUS[170,6]
STRING("ViO")[170,7]
RPAREN[174,11]
INT(7)[174,12]
TIMES[176,1]
RPAREN[178,6]
LBRACK[182,1]
RPAREN[182,2]
LBRACE[182,3]
RBRACK[182,4]
ID(aWAU)[182,5]
LBRACK[182,9]
RPAREN[182,10]
RBRACE[182,11]
LBRACE[182,12]
LBRACK[182,13]
LPAREN[183,1]
INT(6)[183,2]
LBRACE[185,1]
ID(egr)[186,1]
RBRACK[186,4]
RBRACE[186,5]
RPAREN[186,6]
LBRACK[186,7]
RPAREN[186,8]
LBRACE[187,1]
STRING("")[187,2]
INT(19)[187,4]
RBRACK[188,2]
RBRACE[188,3]
STRING("W")[188,4]
PLUS[188,7]
PLUS[188,8]
RPAREN[188,9]
INT(5)[189,3]
PLUS[191,1]
INT(57)[191,2]
INT(80)[192,1]
MINUS[193,1]
INT(40)[193,2]
INT(21)[195,1]
INT(14)[196,1]
LPAREN[197,1]
INT(6048)[198,1]
ID(P)[199,1]
RBRACE[199,2]
INT(974)[199,3]
LPAREN[201,1]
RBRACK[202,1]
INT(897)[202,2]
INT(4731)[203,1]
RBRACE[204,1]
RBRACK[204,2]
ID(j)[204,3]
LBRACE[204,4]
LPAREN[204,5]
LBRACK[204,6]
LBRACK[204,7]
ID(nN)[204,8]
INT(88)[205,1]
PLUS[206,1]
LPAREN[206,2]
INT(3)[206,4]
NIL[207,1]
RBRACE[207,4]
INT(26)[207,5]
RPAREN[208,1]
RBRACK[208,2]
RPAREN[208,3]
INT(33)[208,4]
INT(2215)[209,1]
RBRACE[210,1]
INT(34)[210,2]
RPAREN[211,1]
RBRACE[215,7]
NIL[215,8]
PLUS[215,11]
RBRACK[215,12]
LPAREN[215,13]
STRING("")[215,14]
LBRACE[215,16]
TIMES[216,3]
RBRACE[216,4]
RBRACE[217,1]
INT(48)[217,2]
INT(34)[218,1]
RPAREN[219,1]
LBRACE[219,2]
RBRACK[219,3]
LBRACK[219,4]
ID(nilnil)[219,5]
INT(1)[220,1]
MINUS[221,1]
PLUS[221,2]
ID(Sy)[221,3]
ID(Aj7)[222,1]
INT(722)[224,1]
NIL[225,1]
PLUS[225,8]
LBRACK[225,9]
LBRACE[225,10]
MINUS[225,11]
ID(ib)[225,12]
MINUS[225,15]
INT(871)[225,16]
INT(4)[226,1]
RBRACE[227,1]
RPAREN[227,2]
LBRACK[227,3]
TIMES[227,4]
MINUS[227,5]
RPAREN[227,6]
MINUS[227,7]
INT(680)[227,8]
LPAREN[228,1]
MINUS[228,2]
ID(W247)[228,3]
INT(672)[229,1]
TIMES[230,1]
MINUS[230,2]
PLUS[230,3]
LPAREN[230,4]
INT(1977)[230,5]
RBRACE[231,1]
ID(u)[231,2]
RPAREN[232,1]
LBRACE[232,2]
INT(785)[232,3]
RBRACK[233,1]
PLUS[233,2]
LBRACK[233,4]
LPAREN[233,5]
STRING("BF")[233,6]
MINUS[233,10]
RPAREN[233,16]
INT(95)[233,17]
LBRACK[234,1]
STRING("")[234,2]
NIL[234,4]
STRING("")[234,7]
MINUS[236,4]
LBRACK[236,5]
LBRACE[237,1]
LBRACK[237,2]
PLUS[237,3]
INT(54)[238,1]
LPAREN[239,1]
INT(3)[239,2]
ID(DuzCc)[240,2]
LBRACE[240,8]
ID(nil11)[240,9]
INT(4)[241,1]
RBRACE[243,1]
RPAREN[243,2]
STRING("clnLhfdJEn")[243,3]
STRING("")[243,15]
RPAREN[244,1]
NIL[244,2]
PLUS[244,5]
MINUS[244,6]
TIMES[247,13]
LBRACK[247,14]
INT(9)[247,15]
MINUS[248,1]
NIL[248,2]
PLUS[248,5]
INT(10)[248,6]
STRING("")[249,1]
STRING("orA")[251,1]
RPAREN[251,6]
INT(10)[251,7]
RBRACE[253,1]
RBRACE[253,2]
INT(9)[253,3]
ID(gq)[254,1]
INT(354)[255,1]
INT(4195)[256,1]
RBRACK[257,1]
INT(69)[257,2]
LBRACE[259,1]
RBRACK[259,2]
RBRACK[259,3]
INT(40)[259,4]
ID(d)[260,1]
RBRACE[260,2]
RBRACK[261,1]
PLUS[261,2]
INT(680)[261,3]
INT(76)[262,1]
INT(67)[263,1]
PLUS[264,1]
STRING("")[264,2]
LBRACE[264,4]
RBRACK[264,6]
RPAREN[264,7]
RBRACK[264,8]
PLUS[264,9]
TIMES[264,10]
RPAREN[264,11]
LPAREN[264,12]
INT(2851)[265,1]
MINUS[266,1]
TIMES[266,2]
ID(nil1412)[266,3]
INT(291)[267,1]
STRING("SvK")[268,1]
INT(90)[268,6]
MINUS[269,1]
INT(3314)[269,2]
INT(1)[270,1]
INT(4157)[271,6]
TIMES[272,1]
MINUS[272,2]
RBRACE[272,3]
ID(Ycod)[272,4]
TIMES[272,8]
RPAREN[272,9]
LBRACE[272,10]
INT(9792)[272,11]
LBRACK[273,1]
RBRACE[273,2]
INT(4)[273,3]
INT(8)[275,1]
RPAREN[276,1]
STRING("")[276,2]
RBRACK[276,4]
STRING("KTRo")[276,5]
NIL[278,1]
PLUS[278,4]
STRING("")[278,6]
LPAREN[278,8]
RBRACK[278,9]
MINUS[278,10]
TIMES[278,11]
STRING("JvF")[278,12]
LPAREN[278,17]
RPAREN[278,18]
RPAREN[278,23]
RBRACK[278,24]
MINUS[278,29]
STRING("Sz")[278,30]
INT(40)[278,34]
RPAREN[279,1]
LPAREN[279,2]
NIL[279,3]
RPAREN[279,6]
LPAREN[279,7]
INT(5552)[279,8]
PLUS[280,1]
RBRACK[280,2]
INT(566)[280,3]
RPAREN[282,1]
LBRACE[282,2]
TIMES[282,4]
TIMES[284,1]
INT(14)[284,2]
RPAREN[285,1]
INT(574)[286,1]
ID(nil395)[287,1]
RPAREN[288,1]
MINUS[288,2]
LPAREN[288,3]
MINUS[288,4]
RBRACE[288,6]
STRING("")[288,7]
RPAREN[288,9]
RBRACE[288,10]
INT(58)[295,13]
MINUS[296,1]
RPAREN[296,2]
LPAREN[296,3]
PLUS[296,4]
INT(4183)[296,5]
PLUS[298,1]
RBRACE[298,2]
NIL[300,1]
RPAREN[300,4]
INT(6442)[300,5]
ID(NS)[301,1]
RBRACE[301,3]
STRING("")[301,4]
INT(9927)[301,6]
RBRACE[302,1]
RBRACK[302,2]
RPAREN[302,3]
RPAREN[302,4]
INT(1)[302,5]
RBRACE[303,1]
LPAREN[303,2]
LBRACE[303,3]
RBRACE[303,4]
LPAREN[303,5]
RBRACE[303,6]
RBRACE[303,7]
MINUS[303,8]
STRING("zEttCf")[303,9]
INT(77)[305,7]
MINUS[306,1]
TIMES[306,2]
MINUS[306,3]
TIMES[306,4]
TIMES[306,11]
LBRACK[306,12]
ID(z)[306,13]
STRING("")[309,8]
INT(66)[309,10]
LBRACK[310,1]
ID(nil953)[310,2]
INT(1)[311,1]
ID(nil18)[312,1]
MINUS[313,1]
ID(B90vm)[313,2]
RBRACK[313,7]
INT(8454)[313,8]
ID(K4)[314,1]
INT(400)[315,1]
LBRACK[316,1]
ID(nil1776)[316,2]
INT(1234)[317,1]
INT(288)[318,1]
LBRACE[319,1]
RBRACE[319,2]
MINUS[320,1]
LBRACE[320,2]
PLUS[320,3]
ID(L4o4)[320,4]
MINUS[320,8]
TIMES[320,9]
INT(6)[321,1]
LBRACE[322,1]
LBRACE[323,3]
LPAREN[326,9]
INT(980)[326,10]
RBRACK[327,1]
LPAREN[328,5]
LBRACE[329,4]
INT(6412)[330,1]
STRING("")[332,1]
MINUS[332,3]
LPAREN[333,1]
INT(326)[333,3]
STRING("")[334,1]
RPAREN[334,3]
RBRACK[334,4]
RBRACE[334,5]
LPAREN[334,6]
PLUS[334,7]
LBRACE[334,8]
TIMES[335,1]
LBRACE[335,2]
ID(nile1)[335,3]
TIMES[335,8]
RBRACE[335,9]
RBRACE[335,10]
ID(nil2)[335,11]
LBRACK[337,1]
INT(1278)[337,2]
LBRACE[340,3]
INT(32)[340,4]
LBRACE[341,2]
RPAREN[341,3]
LPAREN[341,4]
RBRACK[341,5]
RBRACK[341,6]
RPAREN[341,7]
RBRACK[341,8]
MINUS[341,9]
MINUS[341,10]
MINUS[341,11]
INT(237)[341,12]
LPAREN[343,10]
INT(62)[343,11]
RBRACK[344,1]
LPAREN[345,6]
PLUS[345,7]
ID(Zkj5s73)[345,9]
MINUS[346,1]
TIMES[346,2]
INT(8)[346,3]
LBRACK[348,1]
LPAREN[348,2]
RPAREN[348,3]
INT(41)[348,4]
STRING("byXOxJRvARMGbyY")[349,1]
ID(x)[349,18]
MINUS[349,19]
INT(28)[350,1]
STRING("")[351,1]
INT(4)[352,1]
RBRACK[355,1]
INT(0)[355,2]
PLUS[356,1]
INT(73)[356,9]
TIMES[357,2]
INT(70)[357,3]
LBRACE[358,1]
STRING("HKhuI")[358,2]
INT(362)[358,9]
INT(123)[359,1]
LBRACK[363,8]
PLUS[363,9]
PLUS[364,1]
LPAREN[366,5]